recipe/build.sh

```diff
@@ -4,20 +4,52 @@
 cp -r ${BUILD_PREFIX}/share/libtool/build-aux/config.* ./config
 
 # avoid absolute-paths in compilers
-# ... this is required that pg_config provides more sane
-# configuration in different environment
+# ... this is required that pg_config provides more sane configuration in different environment
 export CC=$(basename "$CC")
 export CXX=$(basename "$CXX")
 export FC=$(basename "$FC")
 
-./configure \
-    --prefix=$PREFIX \
-    --with-readline \
-    --with-libraries=$PREFIX/lib \
-    --with-includes=$PREFIX/include \
-    --with-openssl \
-    --with-gssapi  \
-    PG_SYSROOT="undefined"
+if [ "${build_platform}" == "linux-s390x" ]; then
+  ./configure \
+      --prefix=$PREFIX \
+      --with-readline \
+      --with-libraries=$PREFIX/lib \
+      --with-includes=$PREFIX/include \
+      --with-openssl \
+      --with-gssapi  \
+      --with-icu \
+      --with-libxml \
+      --with-libxslt \
+      --with-lz4 \
+      --with-zstd \
+      --with-uuid=e2fs \
+      --with-system-tzdata=$PREFIX/share/zoneinfo \
+      PG_SYSROOT="undefined"
+else
+  ./configure \
+      --prefix=$PREFIX \
+      --with-readline \
+      --with-libraries=$PREFIX/lib \
+      --with-includes=$PREFIX/include \
+      --with-openssl \
+      --with-gssapi  \
+      --with-icu \
+      --with-ldap \
+      --with-libxml \
+      --with-libxslt \
+      --with-lz4 \
+      --with-zstd \
+      --with-uuid=e2fs \
+      --with-system-tzdata=$PREFIX/share/zoneinfo \
+      PG_SYSROOT="undefined"
+fi
```

Comment:
```suggestion
EXTRA_OPTS=''
if [ "${build_platform}" == "linux-s390x" ]; then
  EXTRA_OPTS='--with-ldap'
fi

./configure \
      --prefix=$PREFIX \
      --with-readline \
      --with-libraries=$PREFIX/lib \
      --with-includes=$PREFIX/include \
      --with-openssl \
      --with-gssapi  \
      --with-icu \
      --with-libxml \
      --with-libxslt \
      --with-lz4 \
      --with-zstd \
      --with-uuid=e2fs \
      --with-system-tzdata=$PREFIX/share/zoneinfo \
      $EXTRA_OPTS \
      PG_SYSROOT="undefined"
```
It's preferred to be explicit like this. It makes it much easier to see what's different on linux-390x.